src/wiki/views/article.py

```diff
@@ -477,10 +457,8 @@ def form_valid(self, form):
         for ancestor in self.article.ancestor_objects():
             ancestor.article.clear_cache()
 
-        old_path = self.urlpath.parent.path
```

Comment:
This was actually the path of the parent, just a bit mindfucking when trying to understand the loop furtherdown :)